client/js/components/CardEventSmall/CardEventSmall.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  Image,
+  ImageBackground,
+  TouchableOpacity,
+} from 'react-native';
+import {RatingScore00} from '../RatingScore';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const CardEventSmall = ({eventInfo, index}) => {
```

Comment:
Please choose semantic names for your classes, variables and functions.

Reply:
It's a very semantic name for the small-card-event at home screen

![image](https://user-images.githubusercontent.com/20639360/70753455-a508da80-1ce9-11ea-8c38-a0a9d7d69d97.png)